pages/settings.js

```diff
@@ -0,0 +1,88 @@
+                            <input type='text'/>
+                        </span>
+                        <span className={settingsStyle.inputSpanEmojie}>
+                            <p className = {settingsStyle.nameField}>Like + Message:</p>
+                            <input type='text'/>
+                        </span>
+                        <span className={settingsStyle.inputSpanEmojie}>
+                            <p className = {settingsStyle.nameField}>Report:</p>
+                            <input type='text'/>
+                        </span>
+                    </div>
+                    <div className={settingsStyle.buttonsDiv}>
+                        <h2>Buttons:</h2>
+                        <span className={settingsStyle.inputSpanButton}>
+                            <p className = {settingsStyle.nameField}>Account settings:</p>
+                            <input type='text'/>
+                        </span>
+                        <span className={settingsStyle.inputSpanButton}>
+                            <p className = {settingsStyle.nameField}>Menu:</p>
+                            <input type='text'/>
+                        </span>
+                        <span className={settingsStyle.inputSpanButton}>
+                            <p className = {settingsStyle.nameField}>Back:</p>
+                            <input type='text'/>
+                        </span>
+                        <span className={settingsStyle.inputSpanButton}>
+                            <p className = {settingsStyle.nameField}>Start search:</p>
+                            <input type='text'/>
+                        </span>
+                        <span className={settingsStyle.inputSpanButton}>
+                            <p className = {settingsStyle.nameField}>Edit form:</p>
+                            <input type='text'/>
+                        </span>
+                        <span className={settingsStyle.inputSpanButton}>
+                            <p className = {settingsStyle.nameField}>Restart form:</p>
+                            <input type='text'/>
+                        </span>
+                        <span className={settingsStyle.inputSpanButton}>
+                            <p className = {settingsStyle.nameField}>Edit form field:</p>
+                            <input type='text'/>
+                        </span>
+                        <span className={settingsStyle.inputSpanButton}>
+                            <p className = {settingsStyle.nameField}>Activate account:</p>
+                            <input type='text'/>
+                        </span>
+                        <span className={settingsStyle.inputSpanButton}>
+                            <p className = {settingsStyle.nameField}>Deactivate account:</p>
+                            <input type='text'/>
+                        </span>
+                    </div>
+                </div>
+            </form>
+
```

Comment:
When you submit code for review, make sure to check it on your own first to spot anything unusual: unwanted committed changes, missing spacing, grammatical errors or extra lines.

Here you have an extra empty line, what for?